react.mdx

```diff
@@ -174,4 +233,39 @@ React requires Javascript to show the content on the page. The problem is that t
 
 ## Questions
 
-<img src={Questions} />
+<img style={{ height: '50%' }} src={Questions} />
+
+---
+
+<LiveCode
+  title="Hooks version"
+  code={`
+  // ------ Functional ------
+  const Title = (props) => (
+    <h3 style={{ color: 'palevioletred' }}>
+      {props.title}
+    </h3>
+  );
+  // ---- Functional with Hooks ----
+  const Likes = () {
+    const [likes, setLikes] = useState(0)
```

Comment:
I wasn't able to get `useState` working in the demo (but I'm investigating a `transforms: { moduleImport: false }` error / instruction locally)